remote_challenge_evaluation/remote_evaluation_util.py

```diff
@@ -0,0 +1,58 @@
+
+class RemoteEvaluationUtil:
+    def __init__(self):  
+        self.auth_token = ""  # Go to EvalAI UI to fetch your auth token
+        self.evalai_api_server = ""  # For staging server, use -- https://staging.eval.ai; For production server, use -- https://eval.ai
+        self.queue_name = ""  # Please email EvalAI admin (team@cloudcv.org) to get the queue name
+        self.challenge_pk = ""  # Please email EvalAI admin (team@cloudcv.org) to get the challenge primary key
```

Comment:
These variables were being used in `main.py` earlier with `evalai`. Since we added `evalai` as an attribute now, I am directly using these variables in the `__init__` method.